docs/pages_en/advanced/helm/deploy_process/external_dependencies.md

```diff
@@ -0,0 +1,35 @@
+---
+title: External dependencies
+permalink: advanced/helm/deploy_process/external_dependencies.html
+---
+
+If you want to make one release resource depend on another you can change its [deployment order]({{ "/advanced/helm/deploy_process/deployment_order.html" | true_relative_url }}) to make the latter deploy only after the former is deployed successfully. But what if we want to make a release resource depend on a resource that is not managed as part of our release, or even not managed by werf at all (e.g. created by some operator)?
+
+[`<name>.external-dependency.werf.io/resource`]({{ "/reference/deploy_annotations.html#external-dependency-resource" | true_relative_url }}) annotation can define an external dependency on another resource. Until the defined external dependency is created and ready, the annotated resource will not be deployed.
+
+Example:
+```yaml
+kind: Deployment
+metadata:
+  name: app
+  annotations:
+    secret.external-dependency.werf.io/resource: secret/dynamic-vault-secret
+```
+
+In this example the `app` deployment will not be deployed and will wait until Secret `dynamic-vault-secret` is created. Lets assume that `dynamic-vault-secret` is created by the operator from your Vault instance and is not managed by werf.
+
+Another example:
```

Comment:
```suggestion
In the example above, werf will wait for the `dynamic-vault-secret` to be created before proceeding to deploy the `app` deployment. Let's now assume that `dynamic-vault-secret` is created by the operator from your Vault instance and is not managed by werf.

Here is a relevant example:
```

Reply:
Second example is not related to the first one.

This
```
Lets assume that `dynamic-vault-secret` is created by the operator from your Vault instance and is not managed by werf.
```
was about the first example.

Reply:
Rephrased: **We** assume that `dynamic-vault-secret` is created by the operator from your Vault instance...